lib/balrog/routes_middleware.rb

```diff
@@ -21,7 +21,10 @@ def initialize(app)
 
   def call(env)
     unless authenticated?(env['rack.session']['balrog'])
-      html = ApplicationController.renderer.render 'balrog/gate', layout: 'balrog'
+      view_locals = {
+        current_path: env["SCRIPT_NAME"]
```

Comment:
What's this?

Reply:
At one point, I needed to manually specify the return destination, but now we don't. 

I will remove this code. Thanks 🙏